INFO:kivy:[Logger      ] Record log in C:\Users\recy.MOBICAPL\.kivy\logs\kivy_17-10-16_10.txt
INFO:kivy:[Kivy        ] v1.9.2.dev0, git-04b1c58, 20170417
INFO:kivy:[Python      ] v3.6.1 (v3.6.1:69c0db5, Mar 21 2017, 17:54:52) [MSC v.1900 32 bit (Intel)]
INFO:kivy:[Factory     ] 194 symbols loaded
INFO:kivy:[Image       ] Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
INFO:kivy:[OSC         ] using <thread> for socket
INFO:kivy:[Window      ] Provider: sdl2
INFO:kivy:[GL          ] Using the "OpenGL" graphics system
INFO:kivy:[GL          ] GLEW initialization succeeded
INFO:kivy:[GL          ] Backend used <glew>
INFO:kivy:[GL          ] OpenGL version <b'4.0.0 - Build 9.18.10.3293'>
INFO:kivy:[GL          ] OpenGL vendor <b'Intel'>
INFO:kivy:[GL          ] OpenGL renderer <b'Intel(R) HD Graphics 4600'>
INFO:kivy:[GL          ] OpenGL parsed version: 4, 0
INFO:kivy:[GL          ] Shading version <b'4.00 - Build 9.18.10.3293'>
INFO:kivy:[GL          ] Texture max size <8192>
INFO:kivy:[GL          ] Texture max units <16>
INFO:kivy:[Shader      ] fragment shader: <b'No errors.'>
INFO:kivy:[Shader      ] vertex shader: <b'No errors.'>
INFO:kivy:[Shader      ] program: <No errors.>
INFO:kivy:[Window      ] auto add sdl2 input provider
INFO:kivy:[Window      ] virtual keyboard not allowed, single mode, not docked
INFO:kivy:[Text        ] Provider: sdl2
INFO:kivy:[Clipboard   ] Provider: winctypes
INFO:kivy:[GL          ] NPOT texture support is available
INFO:kivy:[Base        ] Start application main loop
ERROR:kivy:[Image       ] Error reading file ../cookbook_gui/recipe_images\pepperplate_piersi_kurczaka_w_sosie_borowikowo-serowym.jpg
ERROR:kivy:[Image       ] Error reading file ../cookbook_gui/recipe_images\pepperplate_piersi_kurczaka_w_sosie_borowikowo-serowym.jpg
INFO:kivy:[Base        ] Leaving application in progress...
ERROR:kivy:Wyst¹pi³ b³¹d aplikacji
Traceback (most recent call last):
  File "C:/Users/recy.MOBICAPL/Dropbox/PYTHON/projekt_cookbook/cookbook_gui_win/app_cookbook.py", line 1221, in <module>
    myapp.run()
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\kivy\app.py", line 828, in run
    runTouchApp()
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\kivy\base.py", line 504, in runTouchApp
    EventLoop.window.mainloop()
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\kivy\core\window\window_sdl2.py", line 663, in mainloop
    self._mainloop()
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\kivy\core\window\window_sdl2.py", line 405, in _mainloop
    EventLoop.idle()
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\kivy\base.py", line 339, in idle
    Clock.tick()
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\kivy\clock.py", line 581, in tick
    self._process_events()
  File "kivy\_clock.pyx", line 367, in kivy._clock.CyClockBase._process_events (kivy\_clock.c:7700)
  File "kivy\_clock.pyx", line 397, in kivy._clock.CyClockBase._process_events (kivy\_clock.c:7577)
  File "kivy\_clock.pyx", line 395, in kivy._clock.CyClockBase._process_events (kivy\_clock.c:7498)
  File "kivy\_clock.pyx", line 167, in kivy._clock.ClockEvent.tick (kivy\_clock.c:3483)
  File "C:/Users/recy.MOBICAPL/Dropbox/PYTHON/projekt_cookbook/cookbook_gui_win/app_cookbook.py", line 197, in <lambda>
    Clock.schedule_once(lambda dt: self.on_touch_down(touch, True))
TypeError: on_touch_down() takes 2 positional arguments but 3 were given
